src/gui/transferlistfilterswidget.cpp

```diff
@@ -306,27 +321,21 @@ TrackerFiltersList::~TrackerFiltersList()
         Utils::Fs::forceRemove(iconPath);
 }
 
-void TrackerFiltersList::addItem(const QString &tracker, const QString &hash)
+void TrackerFiltersList::addItem(const QString &tracker, const BitTorrent::InfoHash &hash)
 {
-    QStringList tmp;
-    QListWidgetItem *trackerItem = nullptr;
-    QString host = getHost(tracker);
-    bool exists = m_trackers.contains(host);
+    const QString host {getHost(tracker)};
+    const bool exists {m_trackers.contains(host)};
+    QSet<BitTorrent::InfoHash> &hashes {m_trackers[host]};
```

Comment:
Now `m_trackers` contains `host` even if it doesn't before. Doesn't seem correct to me.

Reply:
Indeed. Fixed now.